docs/INSTALL.md

```diff
@@ -23,8 +23,7 @@ Hosting Rails applications is complicated. Even if you already have Postgres, Re
 
 - [Postgres 13+](https://www.postgresql.org/download/)
 - [Redis 6+](https://redis.io/download)
-- [Ruby 2.7](https://www.ruby-lang.org/en/downloads/) (we recommend 2.7.2)
```

Comment:
Note that I removed the recommendation here to avoid a moving target as patch releases are made.